packages/astro/src/core/encryption.ts

```diff
@@ -0,0 +1,65 @@
+import { base64, decodeHex, encodeHex } from '@oslojs/encoding';
+
+const ALGORITHM = 'AES-GCM';
+
+export async function createKey() {
```

Comment:
Can you please add some documentation that explains what kind of key this function generates?

Reply:
This is the document I followed for the first implementation if it helps: https://gist.github.com/rpivo/2904951ee31ce9146d32321f12c941bd

We can also link to the Next.js source, but it's a bit too spread out to be helpful in the future.